STACpopulator/input.py

```diff
@@ -201,7 +207,7 @@ def __iter__(self) -> Iterator[Tuple[str, MutableMapping[str, Any]]]:
                 if self.prune:  # stop recursive search if requested
                     del dirs[:]
                 col_path = os.path.join(root, self._collection_name)
-                yield col_path, self._load_json(col_path)
+                yield col_path, "", self._load_json(col_path)
```

Comment:
Since item location is now returned, this should be updated properly to do 
`yield self._collection_name, self._load_json(col_path)`

Reply:
Done.